back/engines/commercial/content_builder/app/controllers/content_builder/web_api/v1/content_builder_layouts_controller.rb

```diff
@@ -35,7 +35,7 @@ def content_buildable
           when 'Project'
             Project.find params[:project_id]
           when 'HomePage'
-            HomePage.first
+            nil
```

Comment:
I'm considering it tech debt, but the report builders should also be supported through this endpoint. I don't have a very strong preference, but I think it can be good to explicitly list all the expected content_buildable types (for example, here you can see that the report builders are (probably) not yet supported because they are not a part of the case statement).